dags/exec_in_pod.py

```diff
@@ -16,15 +16,15 @@ def get_pod_name(deployment_name, namespace):
   api_instance = kubernetes.client.CoreV1Api(kubernetes.client.ApiClient(configuration))
 
   try:
-      pod_label_selector="app=" + deployment_name + ", spilo-role=master"
+      pod_label_selector="app=" + deployment_name + ", " + selector
```

Comment:
does this work if you have a `, ` but no `selector`? Perhaps an update for the next PR